test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -251,6 +280,13 @@ func testPass(testConfig *testConfig, props map[string]string) error {
 		if err := addProperties(xmlpath, props); err != nil {
 			return fmt.Errorf("xml add props: %s", err)
 		}
+
+		if testContainer != nil {
+			if err := testContainer.stopAndRemove(); err != nil {
+				// log but do not return error
+				fmt.Fprintf(os.Stderr, "error stopping and removing test container: %s\n", err)
+			}
+		}
```

Comment:
Same as above, we can do this once after all the testsuites have run.
Secondly, we can even skip this part altogether. I do not see any reason to stop the container, since it is intended to remain for the entire lifetime of the tests.

Reply:
fixed in latest revision